src/app/pages/PerpetualPage/components/OrderHistoryTable/OrderHistoryRow.tsx

```diff
@@ -69,11 +62,11 @@ export const OrderHistoryRow: React.FC<OrderHistoryRowProps> = ({ item }) => {
         {typeText}
       </td>
       <td>{item.orderState}</td>
-      <td>{collateralAsset}</td>
+      <td>{getCollateralName(item?.pair?.collateralAsset || Asset.UND)}</td>
```

Comment:
I don't like it, the new asset is completely unnecessary. You can have `Asset.BTCS` as a fallback.